### PR TITLE
CON-298 -- Change mention of version string in doc/

### PR DESCRIPTION
**Pull request by alvinkrebs**
Update the doc/conf.py and doc/release_notes.rst to show new version 1.3.0.

### CODE REVIEW

File: docs/release_notes.rst [hunk 1]
```diff
@@ -39,11 +39,15 @@ repository <https://github.com/rticommunity/rticonnextdds-connector>`__.
    dependencies is not compatible with those versions.
    Node.js versions greater than v18.7.0 are not currently supported.
 
-Version 1.2.2
+Version 1.3.0
+-----------------
+
```
Comment (rkorte) [inline docs/release_notes.rst:44]: @Alxe, do you want to go ahead and add a statement here like we did for Version 1.2.2?

*RTI Connector* 1.3.0 is built on `RTI Connext 7.3.0 <https://community.rti.com/documentation/rti-connext-730>`__.

Or we could wait until we make the other doc updates (but might as well do it now? I get nervous about blank sections in the documentation that we might forget to update ...)

Comment (Alxe) [reply docs/release_notes.rst:44]: Let's do this, we can continue filling the blanks later on.

@alvinkrebs Can you add @rkorte's suggestion to the "Version 1.3.0" section? Namely:
```
Version 1.3.0
-----------------

*RTI Connector* 1.3.0 is built on 
`RTI Connext DDS 7.3.0 <https://community.rti.com/documentation/rti-connext-dds-730>`__.
```

**Review by Alxe — CHANGES_REQUESTED** [review]
While everything looks good, let's add @rkorte's suggestion.

File: docs/release_notes.rst (hunk 1 above)
Comment (alvinkrebs) [reply docs/release_notes.rst:44]: Ok, I've added that comment

**Comment by rkorte** [comment]
How do I see the built docs via GitHub? (I thought Sam showed me once, but I forget.)

**Review by Alxe — APPROVED** [review]

**Comment by alvinkrebs** [comment]
> @alvinkrebs or @Alxe, how do I see the built docs via GitHub? (I thought Sam showed me once, but I forget.)

You can always pop over to https://github.com/rticommunity/rticonnextdds-connector-js/tree/develop/docs and look at any file. But make sure you've chosen the branch you'd like to see.

**Pull request merged**